airbyte-integrations/connectors/source-gnews/unit_tests/test_wait_until_midnight_backoff.py

```diff
@@ -25,6 +25,6 @@
 def test_wait_until_midnight(test_datetime, test_name, now_utc, expected_backoff_time):
     test_datetime.utcnow = Mock(return_value=datetime.strptime(now_utc, DATE_FORMAT))
     response_mock = Mock()
-    backoff_stratery = WaitUntilMidnightBackoffStrategy(options={}, config={})
+    backoff_stratery = WaitUntilMidnightBackoffStrategy(parameters={}, config={})
```

Comment:
I seems like there was another connector like this (see [the commit I added in the branch](https://github.com/airbytehq/airbyte/pull/22974/commits/66e02098036d92efc463309b189ded773d857443))

Reply:
Wait that is a really weird use of `CheckStream`, `source-freshdesk` isn't even a low-code connector so I had to do a double take about what we're doing. For some reason, on a python connector, the latest change swapped out the check logic for the declarative component. That seems like a very questionable decision, granted this change is minor so we can leave it in, but this really shouldn't have even been here in the first place

Reply:
Maybe `source-freshdesk` is not declarative but it uses `CheckStream` which is in the `airbyte_cdk.sources.declarative` package. I guess the issue is not exactly with our changes on the declarative language but more with how `source-freshdesk` is implemented

Reply:
So this shouldn't have passed review in my opinion. A non-low code connector shouldn't be even using declarative components.